docs/core/tutorials/testing-library-with-visual-studio.md

```diff
@@ -165,13 +143,13 @@ Your test run had no failures, but change it slightly so that one of the test me
 
    ![Test Explorer window with failing tests](./media/testing-library-with-visual-studio/failed-test-window.png)
 
-1. In the **Failed Tests** section, select the failed test, `TestDoesNotStartWith`. The **Test Explorer** window displays the message produced by the assert: "Assert.IsFalse failed. Expected for 'Error': false; actual: True". Because of the failure, all strings in the array after "Error" were not tested.
+1. Select the failed test, `TestDoesNotStartWith`. The **Test Explorer** window displays the message produced by the assert: "Assert.IsFalse failed. Expected for 'Error': false; actual: True". Because of the failure, all strings in the array after "Error" were not tested.
```

Comment:
nit: Select the failed test, `TestDoesNotStartWith`, under the **Failed Tests** section - to make it more explicit that there is a separate grouping for failed tests.